Edit Wiki Style Guides (#6300)
w3f/polkadot-wiki: docs/general/contributing.md

:::

Nonetheless, pull requests, discussions, and contributions from the community are encouraged. Active

> community members who demonstrate a record of good contributions may be given written access to the

Reviewer: read/write access are GitHub terms. Can be kept as is. 
https://stackoverflow.com/questions/70538793/remote-write-access-to-repository-not-granted-fatal-unable-to-access

```suggestion
community members who demonstrate a record of good contributions may be given `write` access to the
```

   fixed.
8. Care should be taken to remain as objective and informative as possible. There should be no
   editorializing, and external bias should not be present.
9. We use the Prettier plugin to standardize the style across documents. You can run this on your
   local copy with `npx lint-staged`, but for simplicity, we also have a bot that runs this for us
   in your PRs.

## Style Guides

:::info Wiki General Style Guides

The style guide from the
[Substrate Knowledge Base](https://github.com/substrate-developer-hub/knowledgebase/blob/master/CONTRIBUTING.md#documentation-style)
provides general guidelines about how to write your Wiki contribution. See also
[the Wiki readme file](https://github.com/w3f/polkadot-wiki?tab=readme-ov-file#contributing-to-documentation).

:::

The Polkadot Wiki is powered by [Docusaurus](https://docusaurus.io/). Docusaurus 3 was a major
version upgrade that caused [MDX errors](https://docusaurus.io/docs/3.2.1/migration/v3). As a
general rule, avoid using curly brackets and special characters. If you must use them, for example,
to get the \<-\> symbol, you can escape the error with a `\`.

> A consequence of the Docusaurus 3 upgrade is the removal of the conditional rendering with Kusama

Reviewer: ```suggestion
Following the Docusaurus 3 upgrade, conditional rendering has been disabled for Kusama
```

and system chains. If you need to display text for different chains, use `Tabs` and `TabItem`

> formatting.

Reviewer: It is okay to just say we moved away from conditional rendering. 

It was a custom built feature by us and it is possible to have conditional rendering with Docusaurus 3. My understanding was that it was not worth the effort, given the decision to remove Kusama guide. 

The code for conditional rendering is here:
https://github.com/w3f/polkadot-wiki/blob/master/scripts/injectPlugin.js

Avoid using exclamation marks `!` and informal sentences, which are usually more appropriate for
blog posts. Running a grammar check before submitting your pull request for review can save
reviewers time and speed up the review process.

To ensure consistency across pages, note the following general terms:

- `relay chain` in text and `Relay chain` at the beginning of a sentence.
- `parachain` in text and `Parachain` at the beginning of a sentence.

The Polkadot Wiki has multiple moving parts. To ensure your contribution does not introduce

> compilation errors and that your page is properly rendered in production, we encourage you to load
> the Wiki locally before submitting a pull request for review. Loading the Wiki locally also adds the
> benefit of checking for broken links.

Reviewer: ```suggestion
compilation errors and that your page is properly rendered in production, we encourage you to build and render
the Wiki on your local machine before submitting a pull request for review. Rendering the Wiki locally also adds the
benefit of checking for broken links.
```
